PhysicsTools/SelectorUtils/interface/VersionedIdProducer.h

```diff
@@ -102,16 +105,21 @@ VersionedIdProducer(const edm::ParameterSet& iConfig) {
     edm::LogWarning("IdInformation")
       << idmsg.str();
 
+    
+
     produces<std::string>(idname);
     produces<edm::ValueMap<bool> >(idname);
     produces<edm::ValueMap<float> >(idname); // for PAT
-    produces<edm::ValueMap<unsigned> >(idname);
+    produces<edm::ValueMap<unsigned> >(idname);  
+    produces<edm::ValueMap<unsigned> >(idname+std::string(bitmap_label));
```

Comment:
I believe Ref will work with TTree::Draw if you call through get(). 
Also' non alphanumerics in branch names is used elsewhere so it must be enforced.